cobra/README.md

```diff
@@ -3,41 +3,78 @@
 Cobra provides its own program that will create your application and add any
 commands you want. It's the easiest way to incorporate Cobra into your application.
 
-In order to use the cobra command, compile it using the following command:
+Install the cobra generator with the command `go install github.com/spf13/cobra/cobra`. 
```

Comment:
In CI, we are currently using GOBIN (see https://github.com/spf13/cobra/blob/master/.github/workflows/Test.yml). Might be worth adding a reference to https://golang.org/cmd/go/#hdr-Compile_and_install_packages_and_dependencies here?

Reply:
good idea.